tool/testcase/main.go

```diff
@@ -45,9 +46,17 @@ var command = &cobra.Command{
 					return (*hexutil.Big)(value).String()
 				},
 				"UppercaseFirst": func(network networkx.Network) string {
-					a := []rune(network.String())
-					a[0] = unicode.ToUpper(a[0])
-					return string(a)
+					networkStr := network.String()
+					if networkStr == "vsl" {
+						return "VSL"
+					}
+					words := strings.Split(networkStr, "-")
+					for i, word := range words {
```

Comment:
Is it needed to add a empty string check to avoid any unnecessary processing? For ex. "ethereum---uniswap" or ""(empty string)